java_execution/java_xdsml/plugins/org.eclipse.gemoc.execution.sequential.javaxdsml.ide.ui/src/org/eclipse/gemoc/execution/sequential/javaxdsml/ide/ui/builder/GemocSequentialLanguageBuilder.java

```diff
@@ -30,7 +30,6 @@
 import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
```

Comment:
Shouldn't this class become a generic builder, and not just a "sequential builder"?

Reply:
not yet, I haven't moved the builder in this PR.

you're right a future action should be to also move the builder (+ nature). As most engines would need only this one, this will simplify the development of new engines.

In the case of moccml projects, 2 builders/natures should be used (the generic + the one dedication to the specific additions (while currently, the unique moccml builder is enough as it updates both extension points)

As this change would require to modify many files I plan to do it in a separate PR when this one is ok on the CI and merged.